examples/aio_get_pages_iterator.py

```diff
@@ -14,8 +14,8 @@
 # for example, in Terminal on macOS:  export MERAKI_DASHBOARD_API_KEY=66839003d2861bc302b292eb66d3b247709f2d0d
 api_key = ""
 
-ORGANIZATION_ID = ""
-NETWORK_ID = ""
+ORGANIZATION_ID = "619244948763444192"

```

Comment:
Did you mean to add this org ID?

Reply:
Not really, but they should already be removed in the latest commit.
Could you do a squash merge please? That way they won't stay in the commit history :)